pkg/lightning/backend/local/local.go

```diff
@@ -1446,9 +1449,10 @@ func (local *Backend) newRegionJobWorker(
 		regenerateJobsFn: local.generateJobForRange,
 	}
 	if kerneltype.IsNextGen() {
+		// TODO: add support for TLS.
+		httpClient := &http.Client{}
 		cloudW := &objStoreRegionJobWorker{
-			// TODO fill the cli
-			ingestCli:      nil,
+			ingestCli:      ingestcli.NewClient(local.TiKVWorkerURL, local.pdCli.GetClusterID(ctx), httpClient, local.splitCli),
```

Comment:
we can do `local.pdCli.GetClusterID(ctx)` inside `doImport`, and pass to this method